src/sagemaker/huggingface/llm_utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Functions for generating ECR image URIs for pre-built SageMaker Docker images."""
+from typing import Optional
+
+from sagemaker import image_uris
+from sagemaker.session import Session
+
+
+def _validate_pt_tf_versions(pytorch_version, tensorflow_version):
+    """Validate PyTorch / TensorFlow versions.
+
+    Args:
+        pytorch_version (str): PyTorch version to validate.
+            Required unless ``tensorflow_version`` is provided. List of supported versions:
+            https://github.com/aws/sagemaker-python-sdk#huggingface-sagemaker-estimators.
+        tensorflow_version (str): TensorFlow version to validate.
+            Required unless ``pytorch_version`` is provided. List of supported versions:
+            https://github.com/aws/sagemaker-python-sdk#huggingface-sagemaker-estimators.
+    """
+
+    if tensorflow_version is not None and pytorch_version is not None:
+        raise ValueError(
+            "tensorflow_version and pytorch_version are both not None. "
+            "Specify only tensorflow_version or pytorch_version."
+        )
+    if tensorflow_version is None and pytorch_version is None:
+        raise ValueError(
+            "tensorflow_version and pytorch_version are both None. "
+            "Specify either tensorflow_version or pytorch_version."
+        )
+
+
+def get_huggingface_llm_image_uri(
+    region: Optional[str] = None,
+    version: Optional[str] = None,
+    py_version: Optional[str] = None,
+    instance_type: Optional[str] = None,
+    container_version: Optional[str]=None,
+    pytorch_version: Optional[str] = None,
```

Comment:
why we need pytorch and tensorflow version here?

Reply:
Removed.